ft/src/test/resources/projects/properties-override/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
```

Comment:
Sorry, i missed it before.
it's better to add the latest gradle (if it is compatible) to the tests like this

Reply:
Why not 8.10.2?